source/agora/crypto/Bech32.d

```diff
@@ -134,15 +134,15 @@ public DecodeResult decodeBech32 (in char[] str)
         ubyte c = str[i];
         if (c >= 'a' && c <= 'z') lower = true;
         else if (c >= 'A' && c <= 'Z') upper = true;
-        else if (c < 33 || c > 126)
-            throw new Exception(format("Character '%X' at pos %s outside of valid char range", c, i));
+        else
+            enforce(c >= '!' && c <= '~',
```

Comment:
Slight modification to make it more readable

Reply:
Maybe even better to create constants that define these as boundaries (which I am assuming they are).

Reply:
Well it's `man ascii`